web-console/future-openshift/diagnostics/diagnostics.md

```diff
@@ -0,0 +1,35 @@
+# Downloading Diagnostics
+
+Users can download a diagnostic report of their system and send it to Red Hat from the console.
+
+
+## Help menu
+![dropdown](img/dropdown.png)
+* The "Download Diagnostics" option is available from the help menu.
+
+## Modal
+![modal](img/modal.png)
```

Comment:
@bmignano It seems like we will have a scaling problem with this checklist since users could have a lot of nodes. Can you consider another way to capture that info from the user?

Reply:
Should the secondary button be cancel here?

Reply:
> @bmignano It seems like we will have a scaling problem with this checklist since users could have a lot of nodes. Can you consider another way to capture that info from the user?

I'm wondering if there should be an "All Nodes" selection too...or maybe this could be the default?

Reply:
@bmignano @lizsurette 
Selecting a partial list of nodes seems to be only relevant for sosreport (which is node specific).  Are we supporting sosreport (I thought CEE said they don't use sosreports)?

For a distributed cluster, you need info from all nodes otherwise you may not have complete information to diagnose the issue.

Also why call out storage diagnostics?  Is this the infrastructure storage, OCS, etc.  Specifically what level of storage information are we referring to?  If storage called out for some reason, why not networking, or are we assuming networking, routing, and other configuration is not an issue?

Reply:
@alimobrem what do you think about the comment from @julienlim? It could be that our requirements for KNI and OCS differ a bit so just curious if we will need to have some extra smarts around this feature to not allow the user to select specific nodes or not...